tests/discovery.spec.coffee

```diff
@@ -257,5 +258,27 @@ describe 'Discoverable Services:', ->
 					expect(privateSsh.subtypes).to.deep.equal([ 'second' ])
 					expect(privateSsh.port).to.equal(2345)
 					expect(privateSsh.protocol).to.equal('tcp')
-
+				.finally ->
 					discoverableServices.unpublishServices()
+
+			it 'should publish single service only to local IPv4 loopback interface', ->
+				# This is a rough check, as by default every Darwin based platform
+				# runs MDNS by default. It is assumed that every other platform (such as
+				# Linux/FreeBSD/Windows) is *not* running a Bonjour based protocol.
+				# If it is, this test will fail as it will not be able to bind to port 5354.
+				if process.platform isnt 'darwin'
+					discoverableServices.publishServices [
+						{ identifier: '_first._sub._ssh._tcp', name: 'First SSH' , port: 1234 }
```

Comment:
You are most definitely a space hound. Spacehound? I do wonder if we should have some linting rules for these... :)